torch/csrc/jit/passes/tensorexpr_fuser.cpp

```diff
@@ -31,90 +31,113 @@ bool isSupported(Node* node) {
+      // case aten::rand_like:
+      "aten::__and__.Scalar(Tensor self, Scalar other) -> Tensor",
+      "aten::__and__.Tensor(Tensor self, Tensor other) -> Tensor",
+      "aten::__or__.Scalar(Tensor self, Scalar other) -> Tensor",
+      "aten::__or__.Tensor(Tensor self, Tensor other) -> Tensor",
+      "aten::__xor__.Scalar(Tensor self, Scalar other) -> Tensor",
+      "aten::__xor__.Tensor(Tensor self, Tensor other) -> Tensor",
+      "aten::__lshift__.Scalar(Tensor self, Scalar other) -> Tensor",
+      "aten::__lshift__.Tensor(Tensor self, Tensor other) -> Tensor",
+      "aten::__rshift__.Scalar(Tensor self, Scalar other) -> Tensor",
+      "aten::__rshift__.Tensor(Tensor self, Tensor other) -> Tensor",
+      "aten::where.self(Tensor condition, Tensor self, Tensor other) -> Tensor",
+      "aten::where.ScalarSelf(Tensor condition, Scalar self, Tensor other) -> Tensor",
+      "aten::where.ScalarOther(Tensor condition, Tensor self, Scalar other) -> Tensor",
+      "aten::where.Scalar(Tensor condition, Scalar self, Scalar other) -> Tensor",
+      "aten::where(Tensor condition) -> Tensor[]",
+      // TODO: enable other min/max variants, operators that can be both
+      // elementwise or reductions:
+      "aten::min.other(Tensor self, Tensor other) -> Tensor",
+      "aten::max.other(Tensor self, Tensor other) -> Tensor",
+      // TODO: enable slice, shape inference is not implemented for this op yet
+  };
+
+  if (node->isMemberOf(supported_operator_set)) {
+    return true;
+  }
+
+  // unschematized ops
   switch (node->kind()) {
-    case aten::add:
-    case aten::_cast_Float:
-    case aten::type_as:
-    case aten::sub:
-    case aten::mul:
-    case aten::div:
-    case aten::eq:
-    case aten::ne:
-    case aten::ge:
-    case aten::gt:
-    case aten::le:
-    case aten::lt:
-    case aten::pow:
-    case aten::clamp:
-    case aten::lerp:
-    case aten::log10:
-    case aten::log:
-    case aten::log2:
-    case aten::exp:
-    case aten::erf:
-    case aten::erfc:
-    case aten::fmod:
-    case aten::cos:
-    case aten::sin:
-    case aten::tan:
-    case aten::acos:
-    case aten::asin:
-    case aten::atan:
-    case aten::atan2:
-    case aten::cosh:
-    case aten::sinh:
-    case aten::tanh:
-    case aten::sqrt:
-    case aten::rsqrt:
-    case aten::abs:
-    case aten::floor:
-    case aten::ceil:
-    case aten::round:
-    case aten::trunc:
-    case aten::threshold:
-    case aten::remainder:
     case prim::ConstantChunk:
-    case aten::cat:
     case prim::ListConstruct:
-    case aten::sigmoid:
-    case aten::relu:
-    case aten::addcmul:
-    case aten::neg:
-    case aten::reciprocal:
-    case aten::sum:
-    case aten::expm1:
-    case aten::lgamma:
-    case aten::unsqueeze:
-    case aten::frac:
-    // TODO: uncomment once we can handle rand+broadcasts
-    // case aten::rand_like:
-    case aten::_sigmoid_backward:
-    case aten::_tanh_backward:
```

Comment:
These were the only two ops that I removed - I can reinclude them if wished. I dont think they are generated anywhere. If you look at the sigmoid and tanh backward they are composed of other ops: https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/runtime/symbolic_script.cpp#L786
https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/runtime/symbolic_script.cpp#L164

The corresponding NNC code should probably also be removed but i'm leaving that for a follow up